tests/test_userstory_39.py

```diff
@@ -56,3 +62,87 @@ def test_melding_overlast():
         'object': verblijfsobject['url'],
     })
     assert 'url' in zaak_object
+
+
+    '''
+    Create temporary files to test file uploading
+    '''
+    # Text file (can't be empty or the API throws an error)
+    fd, txt_path = tempfile.mkstemp()
+    with open(txt_path, 'w') as tmp:
+        tmp.write('some additional information')
+
+    # Image (png format)
+    im_format = 'png'
+    im = Image.new('RGB', (512, 512), 'blue') #plain blue square
+    buffered = BytesIO()
+    im.save(buffered, format=im_format)
+
+    '''
+    Upload the files with POST /enkelvoudiginformatieobject (DRC)
+    '''
+
+    text_attachment = drc_client.create_form('enkelvoudiginformatieobject', {
+    'identificatie': uuid.uuid4().hex,
+    'bronorganisatie' : '1',
+    'creatiedatum' : zaak['registratiedatum'],
+    'titel' : 'detailed summary',
+    'auteur' : 'test_auteur',
+    'formaat' : 'txt',
+    'taal' : 'english',
+    }, {
+        'inhoud': open(txt_path, 'rb')
```

Comment:
we don't want to sent binary data - we want to communicate in base64 with the API. That implies sending/reading files needs to be base64

Reply:
done, with the package "django-extra-fields" that I added to the base requirements

Reply:
But the file extension system is still messy since I don't know what kind of files will be sent